rules/manager_test.go

```diff
@@ -570,14 +570,14 @@ func TestStaleness(t *testing.T) {
 	group.Eval(ctx, time.Unix(1, 0))
 	group.Eval(ctx, time.Unix(2, 0))
 
-	querier, err := st.Querier(context.Background(), 0, 2000)
+	querier, err := st.Querier(0, 2000)
 	require.NoError(t, err)
 	defer querier.Close()
 
 	matcher, err := labels.NewMatcher(labels.MatchEqual, model.MetricNameLabel, "a_plus_one")
 	require.NoError(t, err)
 
-	set := querier.Select(false, nil, matcher)
+	set := querier.Select(context.Background(), false, nil, matcher)
```

Comment:
There is already a `ctx` in this test; I think it would be cleaner to use it.
Probably this is true of other tests, but I didn't check them all in detail.

Reply:
Thanks @bboreham - done. I didn't see it since, the corresponding call to `Querier` also didn't use `ctx`. I can't see any other cases in this file at least.